src/test/resources/processExecution/recorder.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as child_process from "child_process";
+import * as events from "events";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+interface ISpawnOptions {
```

Comment:
Sadly I cannot reuse it due to the way it's defined on NodeJS typings.
I missed that I had duplicated it on recording.ts though.
Changed this to import it from recording.ts